app/models/concerns/cable_ready/updatable.rb

```diff
@@ -1,13 +1,12 @@
 # frozen_string_literal: true
 
 require "active_support/concern"
+require "cable_ready/config"
```

Comment:
I'd consider exposing an accessor on the CableReady module similar to `signed_stream_verifier` rather than referencing the debounce adapter through `CableReady.config`. Thoughts?

Reply:
Hmmm only that it wouldn't fit into the current initializer template, which would be kinda weird?

IIRC the signed_stream_verifier is considered private API whereas this ... is not?

Another thing is documentation, which I need to improve

Reply:
Just verified that referencing `config` in multiple places is the strategy I'm using in Turbo Boost libs too. Carry on. 🖖🏻